ibm/service/power/resource_ibm_pi_network_address_group_member_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/power"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccIBMPINetworkAddressGroupMemberBasic(t *testing.T) {
+	name := fmt.Sprintf("tf-nag-name-%d", acctest.RandIntRange(10, 100))
+	cidr := "192.168.1.5/32"
+	nagMemberRes := "ibm_pi_network_address_group_member.network_address_group_member"
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheck(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMPINetworkAddressGroupMemberConfigBasic(name, cidr),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckIBMPINetworkAddressGroupMemberExists(nagMemberRes),
+					resource.TestCheckResourceAttrSet(nagMemberRes, power.Arg_NetworkAddressGroupID),
+					resource.TestCheckResourceAttr(nagMemberRes, power.Arg_Cidr, cidr),
+					resource.TestCheckResourceAttrSet(nagMemberRes, power.Attr_Name),
+				),
```

Comment:
Can you add a test for the member remove in the network-address-group?

Reply:
That is done in the destroy.